roles/kubernetes/node/defaults/main.yml

```diff
@@ -31,6 +31,10 @@ dns_domain: "{{ cluster_name }}"
 
 kube_proxy_mode: userspace
 
+# An experimental dev/test only dynamic volumes provisioner,
+# for PetSets. Works for kube>=v1.3 only.
+kube_hostpath_dynamic_provisioner: "false"
```

Comment:
I put it so bearing in mind backwards compatibility of changes. But True would be nice, indeed